day_3/story_game.py

```diff
@@ -43,38 +45,71 @@ def infinite_query(phrase, valid_answer):
             answer = input(phrase)
         return answer
 
+    @staticmethod
+    def get_fixed_length_string(string: str, max_length: int) -> str:
+        new_string = ''
+        if len(string) > max_length:
+            space = ' '
+            list_from_string = string.split(space)
+            current_max_length = max_length
+            for word in list_from_string:
+                string_part = word + space
+                if len(new_string) + len(string_part) < current_max_length:
+                    new_string += string_part
+                else:
+                    new_string += '\n' + string_part
+                    current_max_length += max_length
+            new_string.rstrip(space)
+        return new_string if new_string else string
+
+    def text_with_borders_printing(
+            self, tp_brd: str, btm_brd: str,
+            text: str = '',
+            action: Callable = None, action_args: tuple = ()
+    ):
+        result = None
+        brd_length = len(tp_brd.split('\n')[0])
+        print(tp_brd)
+        if text:
+            text_processed = self.get_fixed_length_string(text, max_length=brd_length)
+            print(text_processed)
+        if action:
+            result = action(*action_args)
+        print(btm_brd)
+        return result
+
     def play(self):
         start_event = self.story.get('')
         current_event = start_event
+
         while current_event.left_event or current_event.right_event:
-            print(art.TOP_BORDER)
-            print(current_event.text)
-            print(art.BOTTOM_BORDER)
-            left = current_event.left
-            right = current_event.right
-            if right and left:
-                print(f'1:{current_event.left} \n2:{current_event.right}')
-                choice_phrase = 'Enter 1 or 2:'
-                print(art.TOP_BORDER)
-                choice = self.infinite_query(choice_phrase, ['1', '2'])
-                print(art.BOTTOM_BORDER)
-                if choice == '1':
-                    current_event = current_event.left_event
-                elif choice == '2':
-                    current_event = current_event.right_event
-            elif current_event.right_event and current_event.left_event:
+            self.text_with_borders_printing(art.TOP_BORDER, art.BOTTOM_BORDER, text=current_event.text)
+
+            left, left_text, left_event = '1', current_event.left, current_event.left_event
+            right, right_text, right_event = '2', current_event.right, current_event.right_event
+
+            if right_text and left_text:
+                print(f'{left}:{current_event.left} \n{right}:{current_event.right}')
+                choice_phrase = f'Enter {left} or {right}:'
+                choices = [left, right]
+                choice = self.text_with_borders_printing(
+                    art.TOP_BORDER, art.BOTTOM_BORDER, action=self.infinite_query, action_args=(choice_phrase, choices)
+                )
```

Comment:
Since all of these arguments are constants, I think we can make a separate helper function, e.g.:
```def ask_choice():
    return self.text_with_borders_printing(...)
```

Although I have no strong opinion on this one, so leaving it up to you